google/api_core/client_options.py

```diff
@@ -70,6 +70,7 @@ class ClientOptions(object):
         scopes (Optional[Sequence[str]]): OAuth access token override scopes.
         api_key (Optional[str]): Google API key. ``credentials_file`` and
             ``api_key`` are mutually exclusive.
+        audience (Optional[str]): The desired API audience.
```

Comment:
Please could you include an example string to show the correct format?

Reply:
Done